flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -114,10 +114,11 @@ function stop_kubernetes {
     fi
 }
 
-function debug_copy_and_show_logs {
+function debug_and_show_logs {
     echo "Debugging failed Kubernetes test:"
     echo "Currently existing Kubernetes resources"
     kubectl get all
+    kubectl describe all
```

Comment:
If this also doesn't help, I used these commands for debugging the k8s the last time it was unstable:

```
kubectl get pods -o json -n kube-system
kubectl get pods -o json
kubectl get events -o json
kubectl get deployments -o json
kubectl describe pods
kubectl describe nodes
kubectl get nodes -o json
```